test/functional/rpc_net.py

```diff
@@ -249,11 +249,17 @@ def test_addnode_getaddednodeinfo(self):
         added_nodes = self.nodes[0].getaddednodeinfo()
         assert_equal(len(added_nodes), 1)
         assert_equal(added_nodes[0]['addednode'], ip_port)
+        # check that filtering by node works
```

Comment:
nit: How about making this comment a log message instead?

https://github.com/bitcoin/bitcoin/blob/master/test/functional/README.md#general-test-writing-advice

> Instead of inline comments or no test documentation at all, log the comments to the test log, e.g. self.log.info('Create enough transactions to fill a block'). Logs make the test code easier to read and the test logic easier [to debug](https://github.com/bitcoin/bitcoin/blob/master/test/README.md#test-logging).

Also, since we're touching this function, it might be reasonable to add a commit to convert the surrounding comments of this function to log messages.  Any objections from others?

Reply:
It seems good to me, I'll address it.

Reply:
Done.